src/main/java/org/opensearch/knn/index/util/IndexUtil.java

```diff
@@ -412,4 +412,17 @@ private static Map<String, Version> initializeMinimalRequiredVersionMap() {
         }
         return Collections.unmodifiableMap(versionMap);
     }
+
+    /**
+     * Tell if it is byte index or not
+     *
+     * @param knnEngine knn engine associated with an index
+     * @param parameters parameters associated with an index
+     * @return true if it is binary index
+     */
+    public static boolean isByteIndex(KNNEngine knnEngine, Map<String, Object> parameters) {
+        return KNNEngine.FAISS == knnEngine
```

Comment:
But we have byte index for Lucene right?
Shouldn't this method be `isFaissByteIndex`?